apps/vanilla/src/pages/index.html

```diff
@@ -6,11 +6,136 @@
     <link rel="stylesheet" href="./index.css" />
     <title>Vite App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
-    <script type="module" src="./index.ts"></script>
+  <body class="container">
+    <header class="header font-logo">📚 Books app</header>
+    <!-- Main Section -->
+    <section class="main-section">
```

Comment:
This should be `main` as I mention above
```suggestion
    <main class="main">
```